style.css

```diff
@@ -1,7 +1,89 @@
 html {
-  font-size: 10px;
+  font-size: 12px;
 }
-title {
-  background-color: #3E3C3C;
-  color: blue;
+
+@media only screen and (max-width: 1000px) {
+  .wrapper {
+    max-width: 500px;
+    margin: 0 auto;
+    padding: 50px;
+    background-color: #1c1a19;
```

Comment:
All the css styling as at this point is suppose to be a for mobile version so there is no need to use media queries. Moreover, the dimensions of your design is different from that of the figma as shown below. Please try as much as possible to conform to the figma designs

Your design

![Screenshot from 2022-09-24 03-11-04](https://user-images.githubusercontent.com/65233860/192077761-ce05acdd-ee2d-494e-9bbe-08bb5c854072.png)


Figma design

![Screenshot from 2022-09-24 03-17-54](https://user-images.githubusercontent.com/65233860/192077814-7bfa7ad1-6471-4f5c-be87-ca133eaeb30d.png)